Neos.Media/Classes/Domain/Model/ThumbnailGenerator/ImageThumbnailGenerator.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Because we have empty lines there in all other files and it's default formatting